src/renderer/base/renderer.cpp

```diff
@@ -862,7 +862,7 @@ void Renderer::_PaintOverlay(IRenderEngine& engine,
         // Set it up in a Viewport helper structure and trim it the IME viewport to be within the full console viewport.
         Viewport viewConv = Viewport::FromInclusive(srCaView);
 
-        for (auto srDirty : engine.GetDirtyArea())
+        for (SMALL_RECT srDirty : engine.GetDirtyArea())

```

Comment:
just so long as we're sure!

Reply:
I'm sure. But I'm hoping to update several of the draw methods and the `IRenderEngine` interface quickly in the next PR (for DX) to use `til::rectangle` the whole way through so we're not thinking about this anymore.